app/helpers/students_query_helper.rb

```diff
@@ -1,6 +1,39 @@
 require 'digest'
 
 module StudentsQueryHelper
+  INCLUDE_FOR_STUDENTS = [
```

Comment:
Maybe `Student.column_names.map(&:to_sym) - [:whatever_you_want_to_exclude]`?  Similar to the comment in https://github.com/studentinsights/studentinsights/pull/1445#issuecomment-363300279, I think this depends on whether we want this to be "a `Student` object with one or two things missing" or we want this to be "exactly the partial view of a `Student` that we want to show in the product." 

Based on your PR description, is the idea here "everything but the address and contact info?"

Reply:
Ooooh, I like the `Student.column_names` idea! 

And yep, "everything but the address and contact info." But mostly because those are the two fields I could find that aren't consumed by the school overview page. Giving it a second look, we might want to exclude enrollment_status also. And if you spot more fields that aren't used in the overview page let me know. 🔎 🔎